config/stations.yml

```diff
@@ -1,48 +1,64 @@
 walthamstow_central:
   southbound: 37
   northbound: 30
+  class: interchange
 blackhorse_road:
   southbound: 123
   northbound: 128
+  class: interchange
 tottenham_hale:
   southbound: 195
   northbound: 206
+  class: interchange
 seven_sisters:
   southbound: 329
   northbound: 492
+  class: interchange
 finsbury_park:
   southbound: 893
   northbound: 870
+  class: interchange
 highbury_and_islington:
   southbound: 1035
   northbound: 1026
+  class: interchange
 kings_cross_st_pancras:
   southbound: 1193
   northbound: 1174
+  class: normal
```

Comment:
You're right actually - I was going by this https://tfl.gov.uk/tube/route/victoria/